pkg/router/crl/crl.go

```diff
@@ -506,3 +516,15 @@ func makeStagingDirectory() (string, error) {
 	}
 	return stagingDirName, nil
 }
+
+func GetCRLsUpdated() bool {
+	crlsMutex.Lock()
+	defer crlsMutex.Unlock()
+	return crlsUpdated
+}
+
+func SetCRLsUpdated(value bool) {
```

Comment:
Would it make sense to remove the possibility to set updated to `false`? Taking into account the fact that we want the CRLs to be fully present only at startup.
```suggestion
func SetCRLsUpdated() {
```

Reply:
I think that's reasonable. I've updated this to include that change :+1: